builders/xcm/fees.md

```diff
@@ -0,0 +1,230 @@
+- **`ReserveAssetDeposited(assets: MultiAssets)`** - accrues the asset(s) which have been received into the sovereign account of the local consensus system into holding
+- **`ClearOrigin`** - clears the origin to ensure that later instructions cannot command the authority of the original origin. Useful in cases where the instructions are being relayed from an untrusted source, as often is the case with `ReserveAssetDeposited`
+- **`BuyExecution(fees: MultiAsset, weight_limit: Option<Weight>)`** - takes asset(s) from holding to pay for execution fees. The amount that is actually used is determined by the interpreting system. If the fees are more than the specified amount then the instruction will result in error. You can optionally provide the amount of weight to be purchased. If the given amount of weight is lower than the estimated weight, an error will be thrown
+- **`DepositAsset(assets: MultiAssetFilter, max_assets: Compact, beneficiary: MultiLocation`** - removes asset(s) from holding to be sent to a destination account. The maximum assets defines the maximum number of unique assets to remove from holding, any remaining assets stay in holding
+
+## Calculation of Fees for Native Assets {: #calculation-of-fees-native }
+
+Substrate has introduced a weight system that determines how heavy an extrinsic is. When it comes to paying fees, users will pay a transaction fee that is proportionate to the weight of the call that is being made. One unit of weight is defined as one picosecond of execution time.
+
+The following sections will break down how to calculate XCM fees for Polkadot, Kusama, and Moonbeam-based networks. It's important to note that Kusama uses benchmarking and database read and write weight costs to determine the total weight costs for XCM instructions. Whereas Polkadot and Moonbeam-based networks use a fixed weight cost in their calculations. 
+
+There are two databases available in Polkadot and Kusama, RocksDB (which is the default) and ParityDB, both of which have their own associated weight costs for each network. 
+
+### Polkadot {: #polkadot }
+
+As previously mentioned, Polkadot currently uses a fixed amount of weight for all XCM instructions, which is `1,000,000,000` weight units.
+
+Although Polkadot doesn't currently use database weight units to calculate costs, the weight units for database operations are shared here for reference.
+
+|     Database      |    Read    |   Write    |
+|:-----------------:|:----------:|:----------:|
+| RocksDB (default) | 20,499,000 | 83,471,000 |
+|     ParityDB      | 11,826,000 | 30,052,000 |
+
+With the instruction weight cost established, you can calculate the cost of the instruction in DOT. 
+
+In Polkadot, the `ExtrinsicBaseWeight` is set to `85,212,000` which is mapped to 1/10 cents. Where 1 cent is `10^10 / 10,000`. Therefore, the formula for calculating the final fee in DOT looks like this:
```

Comment:
I would add that this formula is for Plank-DOT which is the smallest denomination unit. Also I would resolve the formula so that you get `TotalWeight * CONSTANT [DOT]`